app/views/kondos/show.html.erb

```diff
@@ -1,14 +1,46 @@
 <div class="container">
-  <div class="row justify-content-center">
-    <div class="col-4">
-      <div>
-        <h4><%= @kondo.name %></h3>
-        <h5><%= @kondo.summary %></h5>
-        <span><%= @kondo.prefecture %></span>
-        <span><%= @kondo.user.first_name %></span>
-        <span><%= @kondo.price %></span>
-        <%= link_to "Book now", new_kondo_booking_path(@kondo.id), class:"btn btn-primary"%>
+  <div class="row justify-content-around">
+    <div class="col-lg-6 col-sm-12 p-5 m-2" style="background: white;">
+      <div class="row text-center">
+        <div class="col-lg-4 col-sm-4">
+          <%= cl_image_tag current_user.user_photo, width: "75%",crop: :fill %>
```

Comment:
Hey Hiro that looks great! The only issue is that now I can't access the show page anymore if I am not logged in, not sure but this line might be the culprit as it is looking for current_user... I think this is not what we want because an unauthenticated user should still be able to see the details of a particular kondo before he decides to book it. Is there any particular reason here for using "current user"? Can we use the owner of the kondo instead, and display his picture?